pkg/controller/persistentvolume/deprecated-recycle-api.md

```diff
@@ -0,0 +1,22 @@
+## Deprecation of Persistent Volume Recycler Controller behavior
+The function of the persistent volume recycler will change in future kubernetes releases.  The current implementation will remain and function as is for the short term but expected to be removed or replaced in future releases.  
+
+## Summary
+The current recycler design only allows for user space volume erasure operations.  This is inadequate for control plane or scrubbing that requires elevated privileges.  Future redesign is anticipated and this document serves as warning of future API change. **USE RECYCLE API AT YOUR OWN RISK AS IT MAY BE REMOVED OR SIGNIFICANTLY CHANGED IN FUTURE KUBERNETES RELEASES**.
+
```

Comment:
Can you be a little more explicit about what the limitations are with the current recycler? Perhaps you could walk through a scenario which demonstrates its limitations? Also, why do you we need control plane or elevated privileges? Perhaps you could cover that in your scenario as well.